draft-packages/divider/docs/Divider.stories.tsx

```diff
@@ -28,83 +24,73 @@ export default {
   decorators: [withDesign],
 }
 
-export const DefaultStory = () => (
+export const DefaultStory = args => (
   <Box m={1}>
-    <Divider variant="canvas" />
+    <Divider variant="canvas" {...args} />
   </Box>
 )
-DefaultStory.storyName = "Default (Kaizen Site Demo)"
+DefaultStory.storyName = "Default (Kaizen Demo)"
 
-export const CanvasDivider = () => (
-  <Box m={1}>
-    <Divider variant="canvas" />
-  </Box>
+const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
+  isReversed,
+}) => (
+  <StoryWrapper isReversed={isReversed}>
+    <StoryWrapper.Row rowTitle="Content">
+      <Divider variant="content" isReversed={isReversed} />
```

Comment:
We could possible add a Box wrapper around this with padding to make the story present with a little more spacing?

Reply:
I was also thinking about it, but like you said, it populates the DOM with something extra that isn't necessary. To be honest, it's already a little unclear outside of the example whether the divider comes with margins or not just by looking at it. I'm wondering if it might possibly cause more confusion 🤔 I'd probably lean towards having the sticker sheets be a little more "pure" with less intervention (although it's not a strong opinion, feel free to try and sway me).

Either way, this would probably be one of the components which would benefit from having more stories to showcase usage. I'd leave that to be done separately though (since you'll need to put thought into striking the right balance) :)